Code/Engine/Foundation/Containers/IdTable.h

```diff
@@ -50,7 +50,7 @@ class ezIdTableBase
 
     explicit ConstIterator(const ezIdTableBase<IdType, ValueType>& idTable);
 
-    const ezIdTableBase<IdType, ValueType>& m_idTable;
+    const ezIdTableBase<IdType, ValueType>& m_IdTable;
     IndexType m_uiCurrentIndex; // current element index that this iterator points to.
```

Comment:
Vital build fix? 😁

Reply:
As this is the preparation for the next pr, does it really matter?

Reply:
Of course not.